apps/web/src/modules/auction/components/CurrentAuction/Settle.tsx

```diff
@@ -17,11 +22,18 @@ export const Settle = ({ isEnding }: SettleProps) => {
   const { data: signer } = useSigner()
   const addresses = useDaoStore((state) => state.addresses)
 
+  const { data: paused } = useContractRead({
+    enabled: !!addresses?.auction,
+    address: addresses?.auction,
+    abi: auctionAbi,
+    functionName: 'paused',
+  })
+
   const { config, error } = usePrepareContractWrite({
     enabled: !!addresses?.auction,
     address: addresses?.auction,
     abi: auctionAbi,
-    functionName: 'settleCurrentAndCreateNewAuction',
+    functionName: paused ? 'settleAuction' : 'settleCurrentAndCreateNewAuction',
```

Comment:
oh are you still able to settle a last auction if paused? didn't know that!

Reply:
yes the `settleAuction` function can only be called when the contract is paused and it basically mints the pending nft without starting a new auction.